rfcdiff.js

```diff
@@ -27,7 +27,7 @@ function change_chunk(offset) {
     if (old_chunk) {
         old_chunk.style.outline = "";
     }
-    old_chunk = new_chunk.parentNode.parentNode;
+    old_chunk = new_chunk;
```

Comment:
Go with getElementById().  There wasn't an ID in 1.42, which you based your code on.  I just did the minimal tweaks to your code to make it work with the latest rfcdiff, as the name attributes went away in 1.44.